makiflow/layers/trainable_layers.py

```diff
@@ -283,14 +295,16 @@ def _training_forward(self, X):
     def build(params: dict):
         name = params[BiasLayer.NAME]
         D = params[BiasLayer.D]
-        return BiasLayer(D=D, name=name)
+        trainable = params[BiasLayer.TRAINABLE]
+        return BiasLayer(D=D, trainable=trainable, name=name)
 
     def to_dict(self):
         return {
             BiasLayer.FIELD_TYPE: BiasLayer.TYPE,
```

Comment:
MakiRestorable.FIELD_TYPE. Follow the object oriented design, it is illogical to refer to these constants via the child class. Otherwise the pull request won't be accepted.